fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -509,6 +520,12 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     @VariableMgr.VarAttr(name = QUERY_TIMEOUT)
     private int queryTimeoutS = 300;
 
+    @VariableMgr.VarAttr(name = IS_SET_USE_PAGE_CACHE)
+    private boolean isSetUsePageCache = false;
+
+    @VariableMgr.VarAttr(name = USE_PAGE_CACHE)
+    private boolean usePageCache = false;
+
```

Comment:
Why add two session variables?

Reply:
> Why add two session variables?

![image](https://user-images.githubusercontent.com/45813655/232715674-4086268f-8f56-49f7-a608-d57ce819edf4.png)

Reply:
Use one session variables and set default to true, the effect is the same?

Reply:
> Use one session variables and set default to true, the effect is the same?

updated